### PR TITLE
Refactored, modularized and documented ansible hypervisor playbooks

### PR DESCRIPTION
**Pull request by fbac**
# Description

No fixes on this PR.

Refactored and modularized the ansible playbooks, now it will be a bit more modularized so we can extend it easily.

Added a README.md to document important variables and usage.

## Type of change

Please select the appropriate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [ ] This change is a documentation update

## Testing

Tested in local vm's

**Test Configuration**:

- Versions: 
ansible 2.9+
python3-rpm-4.14+

## Checklist

- [x] I have performed a self-review of my own code
- [x] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not be merged unless it passes CI or there is a comment/update saying what testing was passed.
- [x] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: hack/deploy-hub-local/ansible/tasks/services/chronyd/00-chrony-base.yaml [hunk 1]
```diff
@@ -1,14 +1,15 @@
+---
 - name: Install helpful packages
   package: name={{ item }} state=present
   with_items:
     - chrony
```
Comment (jparrill) [inline hack/deploy-hub-local/ansible/tasks/services/chronyd/00-chrony-base.yaml:5]: Just one package

File: hack/deploy-hub-local/ansible/tasks/base/01-services.yaml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+# TODO: Check if fedora >= 33 or centos = 7|8 so the release in copr exists
+- name: Enable copr repo karmab/kcli
+  command: "dnf copr enable -y karmab/kcli"
+  ignore_errors: true
+  when: os == "fedora"
+
+- name: Install helpful packages
+  package: name={{ item }} state=present
+  with_items:
+    - bash-completion
```
Comment (jparrill) [inline hack/deploy-hub-local/ansible/tasks/base/01-services.yaml:44]: add:
- bind-utils
- httpd-tools

**Review by jparrill — CHANGES_REQUESTED** [review]
Apart of the changes requested, LGTM

File: hack/deploy-hub-local/ansible/tasks/services/chronyd/00-chrony-base.yaml (hunk 1 above)
Comment (fbac) [reply hack/deploy-hub-local/ansible/tasks/services/chronyd/00-chrony-base.yaml]: Changes applied.

**Review by jparrill — APPROVED** [review]
LGTM

**Comment by fbac** [comment]
Added:
- TODO in README.md
- .gitignore to ignore local files named inventory*
- PS1 for root
- resolv.conf and resolv.external.conf now point to google dns by default. In a 2-day step we configure this thoroughly.
- journalSystemMaxUse variable to define journal Max Usage, defaults to 500M.
- Repository EPEL enabled when os=fedora
- packages bind-utils, httpd-tools, joe, ansible, podman, spice-server, spice-protocol, qemu-kvm-ui-spice
- 

Modified:
- Sanitized README.md
- chronyd services now install correctly
- SSH keys now will be retrieved from github using the users in the inventory.

Deleted:
- Old inventory.yaml
- Inventory variables to set ssh keys.

**Pull request merged**